ts/store/reducers/index.ts

```diff
@@ -170,7 +170,8 @@ export function createRootReducer(
                 transactionsRead: state.entities.transactionsRead
               },
               // backend status must be kept
-              backendStatus: state.backendStatus
+              backendStatus: state.backendStatus,
+              crossSessions: state.crossSessions
```

Comment:
Please leave a comment here to note why we're recovering previous crossSessions state